subgraph/package.json

```diff
@@ -22,7 +22,41 @@
+    "generate-gnosis": "mustache config/gnosis.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-goerli": "mustache config/goerli.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-mainnet": "mustache config/mainnet.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-mumbai": "mustache config/mumbai.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-optimism": "mustache config/optimism.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-palm": "mustache config/palm.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-polygon": "mustache config/polygon.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "deploy-hosted-arbitrum": "graph deploy --product hosted-service unlock-protocol/arbitrum-v2",
+    "deploy-hosted-avalanche": "graph deploy --product hosted-service unlock-protocol/avalanche-v2",
+    "deploy-hosted-bsc": "graph deploy --product hosted-service unlock-protocol/bsc-v2",
+    "deploy-hosted-celo": "graph deploy --product hosted-service unlock-protocol/celo-v2",
+    "deploy-hosted-gnosis": "graph deploy --product hosted-service unlock-protocol/gnosis-v2",
+    "deploy-hosted-goerli": "graph deploy --product hosted-service unlock-protocol/goerli-v2",
+    "deploy-hosted-mainnet": "graph deploy --product hosted-service unlock-protocol/mainnet-v2",
+    "deploy-hosted-mumbai": "graph deploy --product hosted-service unlock-protocol/mumbai-v2",
+    "deploy-hosted-optimism": "graph deploy --product hosted-service unlock-protocol/optimism-v2",
+    "deploy-hosted-polygon": "graph deploy --product hosted-service unlock-protocol/polygon-v2",
+    "deploy-studio-arbitrum": "graph codegen && graph build && graph deploy --studio unlock-test-arb",
+    "deploy-studio-avalanche": "graph codegen && graph build && graph deploy --studio unlock-test-avax",
+    "deploy-studio-base-goerli": "graph codegen && graph build && graph deploy --studio unlock-protocol-base-goerli",
+    "deploy-studio-base": "graph codegen && graph build && graph deploy --studio unlock-protocol-base",
+    "deploy-studio-celo": "graph codegen && graph build && graph deploy --studio unlock-protocol-celo",
+    "deploy-studio-gnosis": "graph codegen && graph build && graph deploy --studio unlock-protocol-xdai",
+    "deploy-studio-goerli": "graph codegen && graph build && graph deploy --studio unlock-protocol-goerli",
+    "deploy-studio-mainnet": "graph codegen && graph build && graph deploy --studio unlock-protocol",
+    "deploy-studio-mumbai": "graph codegen && graph build && graph deploy --studio unlock-protocol-mumbai",
+    "deploy-studio-optimism": "graph codegen && graph build && graph deploy --studio unlock-protocol-optimism",
+    "deploy-studio-polygon": "graph codegen && graph build && graph deploy --studio unlock-protocol-polygon"
```

Comment:
dont we want to use a `deploy-hosted` / `deploy-studio` with cli args instead of declaring all networks ? It may be easier to maintain

Reply:
Can definitely do this, just wanted to be very explicit in the beginning, but we could strip them out as long as the slugs are stored for the hosted & studio endpoints.

Reply:
I have removed and left in two commands without the endpoint slug. One command for hosted and one for studio

Reply:
cool, thank you